apps/web/src/components/layout/components/AuthLayout.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Box, createStyles } from '@mantine/core';
+import { ThemeProvider } from '../../../design-system/ThemeProvider';
+
+export default function AuthLayout({ children }: { children: React.ReactNode }) {
+  const { classes } = useStyles();
+
+  return (
+    <ThemeProvider>
+      <div>
+        <div className={classes.wrapper}>
+          <img src="/static/images/logo.png" alt="logo" style={{ maxWidth: 150 }} />
+        </div>
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
+            '@media (max-width: 1200px)': {
+              display: 'none',
+            },
+          }}>
+          <img
+            src="/static/images/notifications/notification_01.png"
+            alt="logo"
+            style={{ position: 'absolute', maxWidth: 400, left: '7%', top: '35%' }}
+          />
+          <img
+            src="/static/images/notifications/notification_02.png"
+            alt="logo"
+            style={{ position: 'absolute', left: '10%', top: '45%', maxWidth: 400 }}
+          />
+          <img
+            src="/static/images/notifications/notification_03.png"
+            alt="logo"
+            style={{ position: 'absolute', left: '7%', top: '55%', maxWidth: 400 }}
+          />
+        </Box>
+
+        {children}
+      </div>
+    </ThemeProvider>
+  );
+}
+
+const useStyles = createStyles((theme) => ({
+  wrapper: {
+    height: '100%',
```

Comment:
![CleanShot 2022-03-24 at 10 35 25](https://user-images.githubusercontent.com/8872447/159875770-21ab1535-a8eb-4abe-9d07-b02c46099500.png)
There is a strange behaviour when the height of the screen is smaller and a scroll area is generated the bg image is not stretched to the end. Maybe worth creating an overflow scroll in this case for the right form section and that it's height will also be 100% height and then overflow: auto; 

What do you think?

Reply:
Sounds good!